pkg/virt-handler/vm.go

```diff
@@ -2173,6 +2173,34 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 		smbios := d.clusterConfig.GetSMBIOS()
 		period := d.clusterConfig.GetMemBalloonStatsPeriod()
 
+		// Find preallocated or thick-provisioned disks
+		var preallocatedVolumes, thickProvisionedVolumes []string
```

Comment:
Is it necessary to keep two separate lists?  What we are really trying to find is a single list of volumes which should have discard passthrough.  Perhaps we should abstract the fact that two different annotations can trigger the condition within a single function.

Reply:
Yes, you're right it's a bit overkill